import_export/utils.py

```diff
@@ -23,3 +27,61 @@ def __enter__(self):
     def __exit__(self, *args):
         if self.using_transactions:
             self.context_manager.__exit__(*args)
+
+
+def diff_lines_to_words(text1, text2):
```

Comment:
Any chance you can write a couple of tests that would give good coverage on these functions? They are quite involved, and Id hate to see a small 🐛  come crawling out of them.

Reply:
Okay. I will try to do it.